File: POOBanco/Python/Dinheiro.py
from Valor import Valor

class Dinheiro(Valor):
    def apresentacao(self):
        print("A %s tem o valor de R$%.2f." % (self.getTipo(), self.getVale()))

    def __init__(self, vale, tipo):
        super().__init__(vale, tipo)

    def __del__(self):
        print("Objeto Dinheiro Destruído com Sucesso!")

File: POOBanco/Java/Dinheiro.java
package poobanco;

public class Dinheiro extends Valor {
    @Override
    public void apresentacao() {
        System.out.printf("A %s tem o valor de R$%.2f.\n", this.getTipo(), this.getVale());
    }
    
    public Dinheiro(float vale, String tipo) {
        super(vale, tipo);
    }
}
